cd .../inotify-tools-4.25.9.0/ipkg-install/opt/bin && \
ln -s inotifywait fsnotifywait && \
ln -s inotifywatch fsnotifywatch
ln: failed to create symbolic link 'fsnotifywait': File exists

---

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -22,8 +22,8 @@ endif
 
 install-exec-hook:
 	cd $(DESTDIR)$(bindir) && \
-	$(LN_S) inotifywait fsnotifywait && \
-	$(LN_S) inotifywatch fsnotifywatch
+	ln -sf inotifywait fsnotifywait && \
+	ln -sf inotifywatch fsnotifywatch
 
 uninstall-hook:
 	rm -f $(DESTDIR)$(bindir)/{fsnotifywait,fsnotifywatch}
